src/main/resources/config/liquibase/changelog/20240530123000_changelog.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                            http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="20240601123000-1" author="yassinsws">
+        <!-- Update the discriminator column to include the new subtype -->
+        <update tableName="iris_sub_settings">
+            <column name="discriminator" value="ingestion"/>
+            <where>discriminator IS NULL</where>
+        </update>
```

Comment:
This is not needed. There are not entries with an empty discriminator. And even if there are they should not be set to "ingestion"